ethereum/p2p/src/main/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/FindNeighborsPacketData.java

```diff
@@ -52,7 +52,7 @@ public static FindNeighborsPacketData readFrom(final RLPInput in) {
     in.enterList();
     final BytesValue target = in.readBytesValue();
     final long expiration = in.readLongScalar();
-    in.leaveList();
+    in.leaveList(true);
```

Comment:
(optional) make leaveList parameter an enum. Much easier to understand what
`in.leaveList(RLPInput.IgnoreRest)` does as opposed to `in.leaveList(true)`

Reply:
I'd say either @RatanRSur's suggestion or at least leave a comment that (true) means to ignore the rest of the list, for all the `leaveList(true)` calls.

Reply:
Third option would be to have two methods - the usual `leaveList` and the new `leaveListSkippingRemainingElements`

Reply:
Changed this to `leaveListLenient()`